packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -21,6 +20,10 @@ export const tooltipShorthandProps: TooltipShorthandProps[] = ['content'];
 
 const mergeProps = makeMergeProps<TooltipState>({ deepMerge: tooltipShorthandProps });
 
+// Style values that are required for popper to properly position the tooltip
+const tooltipBorderRadius = 4; // Update the root's borderRadius in useTooltipStyles.ts if this changes
```

Comment:
can we use proper JSDoc comment or at least comment before token definition ?

Reply:
JSDoc is only used for public/exported things, right? My reasoning for the comments on the same line was to make it blindingly obvious what each constant needs to be kept in sync with.